cuchem/benchmark/scripts/zinc_train_db.py

```diff
@@ -0,0 +1,26 @@
+#! /usr/bin/env python3
+import os
+import sys
+import csv
+from dask import dataframe as dd
+from dask.distributed import LocalCluster, Client
+import sqlite3 as lite
+
+def upload(path='zinc_csv_split/train/*.csv'):
+    print(f'Loading data from {path}...')
+    db = 'sqlite:///zinc_train.sqlite3'
+
+    cluster = LocalCluster(n_workers=4, threads_per_worker=2)
+    client = Client(cluster, asynchronous=True)
+
+    zinc_data = dd.read_csv(path)
+    zinc_data = zinc_data.head()
```

Comment:
@rilango Is this supposed to only put part of the data into the database? Assume this was debugging, if so, we should remove before running.